code/modules/client/preferences_savefile.dm

```diff
@@ -563,6 +564,38 @@
 
 	return TRUE
 
+/datum/preferences/proc/save_loadout_manager()
+	if(!path)
+		return FALSE
+	if(!fexists(path))
+		return FALSE
+	var/savefile/S = new /savefile(path)
+	if(!S)
+		return FALSE
+	loadout_manager = sanitize_loadout_manager(loadout_manager)
+	var/json_loadout_manager = jatum_serialize(loadout_manager)
+	S.cd = "/loadouts"
+	WRITE_FILE(S["loadouts_manager"], json_loadout_manager)
```

Comment:
I assume this is a seperate sav file so we can move it to db or something? or what is this?

Reply:
No, it's the savefile of each player. 
S.cd loadouts creates a new folder specialy for loadouts, but it's the same root folder than gamesettings, character or saved keybinds

Reply:
Ok Apply versioning to it in that case so it can be maintained

Reply:
the versionning is done on the loadouts, but i guess you are right, it should be on the loadout_manager as well